ISISComputingGroup/EPICS-ioc
# Added IOC for Chell CCD100

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/1827

### To test

* Connect to the CCD100 in G17
* Confirm that the IOC PVs write to the device
* Confirm that changing settings on the front panel of the device changes PVs

---

#### Code Review


### Functional Tests


### Final steps
